ui/Document.tsx

```diff
@@ -37,7 +38,7 @@ export function Document({ html, css, scripts, state }: DocumentProps) {
             __html: `window.APP_STATE = { APOLLO_STATE:${JSON.stringify(state.APOLLO_STATE).replace(
               /</g,
               '\\u003c',
-            )}, CONFIG: ${JSON.stringify(state.CONFIG)} };`,
+            )}, CONFIG: ${JSON.stringify(state.CONFIG)}, HEAD: ${JSON.stringify(state.HEAD)} };`,
```

Comment:
Is it really necessary to pass this state? I really don't like having to manually pass state from the server to client, and in this case the hydration pass should generate the same result.

Reply:
Yes. Otherwise the hydration on the client will remove any tags that are sent only from server. I.E title, meta.